skimage/transform/_warps.py

```diff
@@ -97,26 +104,48 @@ def resize(image, output_shape, order=1, mode=None, cval=0, clip=True,
         warn("The default mode, 'constant', will be changed to 'reflect' in "
              "skimage 0.15.")
 
+    if anti_aliasing is None:
+        anti_aliasing = False
+        warn("Anti-aliasing will be enabled by default in skimage 0.15 to "
+             "avoid aliasing artifacts when resampling images.")
+
     output_shape = tuple(output_shape)
-    ndim_out = len(output_shape)
+    output_ndim = len(output_shape)
     input_shape = image.shape
-    if ndim_out > image.ndim:
+    if output_ndim > image.ndim:
         # append dimensions to input_shape
-        input_shape = input_shape + (1, ) * (ndim_out - image.ndim)
+        input_shape = input_shape + (1, ) * (output_ndim - image.ndim)
         image = np.reshape(image, input_shape)
-    elif ndim_out == image.ndim - 1:
+    elif output_ndim == image.ndim - 1:
         # multichannel case: append shape of last axis
         output_shape = output_shape + (image.shape[-1], )
-        ndim_out += 1
-    elif ndim_out < image.ndim - 1:
+    elif output_ndim < image.ndim - 1:
         raise ValueError("len(output_shape) cannot be smaller than the image "
                          "dimensions")
 
-    input_shape = np.asarray(input_shape, dtype=float)
-    dim_scales = input_shape / np.asarray(output_shape)
+    factors = (np.asarray(input_shape, dtype=float) /
+               np.asarray(output_shape, dtype=float))
+
+    if anti_aliasing:
+        if anti_aliasing_stddev is None:
+            anti_aliasing_stddev = np.maximum(0, (factors - 1) / 2)
+            anti_aliasing_stddev[factors <= 1] = 0
```

Comment:
Isn't this line redundant with the one above it?

Reply:
Indeed, fixed.